_posts/2015-05-21-use-docker-to-try-pulp.md

```diff
@@ -12,12 +12,12 @@ categories:
 Using Docker, you can now easily and quickly stand up a demo deployment of Pulp. Excluding the time it takes Docker to download the images, it takes only a few seconds to have a brand new deployment running.
 
     
-    $ wget https://raw.githubusercontent.com/pulp/pulp_packaging/master/dockerfiles/centos/start.sh
+    $ wget https://raw.githubusercontent.com/pulp/packaging/docker/dockerfiles/centos/start.sh

```

Comment:
This line has a carriage return + line feed (CRLF) at the end of it when all of our code only has a line feed (LF). You can't see it in this PR, but I can see it in the `git diff` as a `^M` character. Could that be fixed? You can read more about it [here](https://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it). Maybe use an editor that doesn't use CRLF.